packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -769,19 +769,17 @@ class AppInstance {
 
 /// A [Logger] which sends log messages to a listening daemon client.
```

Comment:
You should at a minimum leave a TODO mentioning that it might make sense to refactor this code.